sdk/tables/azure-data-tables/azure/data/tables/_models.py

```diff
@@ -191,6 +221,14 @@ def _from_generated(cls, generated, **kwargs):  # pylint: disable=unused-argumen
             days=generated.days,
         )
 
+    def __str__(self):
+        # type: () -> str
+        return u"enabled={}, days={}".format(self.enabled, self.days)
```

Comment:
`__str__` is not supposed to return unicode. Must be type `str`

Reply:
Not sure whether the same applied to `__repr__` - would need to check the Py2.7 stdlib ref docs.